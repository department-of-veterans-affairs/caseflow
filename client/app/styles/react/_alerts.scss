
.usa-alert {

  &.fixed {
    position: fixed;
    top: 0;
    margin-top: 0;
    z-index: 999;
    left: 50%;
    transform: translate(-50%, 0);
  }

  // Remove after USWDS V.2.0
  &.usa-alert-slim {
    background-size: 4rem;

    .usa-alert-body {
      padding-left: 3.5rem;
    }

    .usa-alert-text {
      font-size: 2rem;
    }
  }

  .usa-alert-text {
    line-height: 1.5;
  }
}
