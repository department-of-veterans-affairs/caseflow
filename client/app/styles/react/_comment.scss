.comment-control-button-container {
  display: flex;
  width: 100%;
  padding: 3px 0;
  justify-content: space-between;
}

.comment-save-button-container {
  display: inline-block;
  width: 100%;
  margin-top: 5px;
  margin-bottom: 17px;
}

.comment-control-button {
  padding: 0;

  &.horizontal {
    text-align: left;
  }
}

.comment-control-button-divider {
  padding: 0 10px;
}

.comment-container-selected {
  box-shadow: 3px 3px 5px $color-gray-light;
}

.comment-container {
  word-wrap: break-word;
  word-break: break-word;
  background-color: $color-white;
  padding: 15px;
  cursor: pointer;

  &:not(.comment-horizontal-container) {
    margin-bottom: 15px;
    border-radius: 3px;
    border: 1px solid $color-gray-lighter;
  }

  &.comment-horizontal-container {
    padding: 25px 25px 25px 40px;
  }
}

// Max width is enforced so the size of this comment box
// fits with existing textareas in style guide
.comment-size-container {
  max-width: 430px;

  .cf-form-textarea {
    margin-bottom: 20px;
  }
}

.comment-horizontal-container {
  display: inline-block;
  width: 100%;
}

.comment-horizontal-spacing {
  margin-right: 49px;
}

.commentIcon-container {
  position: absolute;
  z-index: 10;
  // These dimensions need to match the comment icon SVG dimensions.
  width: 40px;
  height: 40px;
}

