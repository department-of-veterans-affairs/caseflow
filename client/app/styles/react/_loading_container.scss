.loadingContainer-positioning {
  position: absolute;
  width: 100%;
  height: 100%;
}

.loadingContainer-table {
  width: 100%;
  height: 100%;
  display: table;
}

.loadingContainer-table-cell {
  text-align: center;
  display: table-cell;
  vertical-align: middle;
}

.loadingContainer-content {
  position: relative;
  z-index: 1;

  iframe {
    width: 100%;
    height: 80vh;

    &.form9Viewer {
      height: 500px;
    }
  }
}

.loadingContainer-container {
  position: relative;
  overflow: hidden;
}

.cf-loading-button-text {
  display: inline-block;
  margin-right: 10px;
  line-height: 0;
  position: relative;
}

.cf-loading-message {
  .loading-message-text {
    display: inline-block;
    font-style: italic;
    font-weight: bolder;
  }

}

.cf-loading-button-symbol {
  display: inline-block;
}
