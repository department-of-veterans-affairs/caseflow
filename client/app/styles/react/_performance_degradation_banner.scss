.usa-banner {
  background-color: $color-gold-lightest;
  text-align: center;
}

.usa-banner-inner {
  padding: .8em;
}

.banner-icon {
  display: inline-block;
  vertical-align: text-top;
  // height: 25px;
}

.banner-text {
  padding-left: 6px;
}
