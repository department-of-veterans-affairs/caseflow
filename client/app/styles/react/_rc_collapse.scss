@mixin common() {

  width: 0;
  height: 0;
  font-size: 0;
  line-height: 0;
}

@mixin right($w, $h, $color) {

  border-top: $w solid transparent;
  border-bottom: $w solid transparent;
  border-left: $h solid $color;
}

@mixin bottom($w, $h, $color) {

  border-left: $w solid transparent;
  border-right: $w solid transparent;
  border-top: $h solid $color;
}

// scss-lint:disable IdSelector
#arrow {
  @include common();
  @include right(3px, 4px, $color-gray);
  @include bottom(3px, 4px, $color-gray);
}

.rc-collapse {

  &-anim-active {
    transition: height .2s ease-out;
  }

  & > &-item {
    &:first-child {
      border-top: none;
    }
  }

  & > &-item-disabled > .rc-collapse-header {
    cursor: not-allowed;
    color: $color-gray-light;
    background-color: $color-gray-lighter;
  }

  &-content {
    overflow: hidden;

    &-inactive {
      display: none;
    }
  }

  &-item:last-child {
    > .rc-collapse-content {
      border-radius: 0 0 3px 3px;
    }
  }

  & > &-item-active {
    > .rc-collapse-header {

      .arrow {

        #arrow > {
          @include bottom(3px, 4px, $color-gray);
        }
      }
    }
  }
}

.usa-accordion-content {
  &:not([aria-hidden]) {
    position: relative;
    left: 0;
  }
}
