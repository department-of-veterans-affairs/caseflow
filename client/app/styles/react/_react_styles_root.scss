@import '_comment.scss';
@import '_search_bar.scss';
@import '_performance_degradation_banner.scss';
@import '_rc_collapse.scss';
@import '_loading_container.scss';
@import '_select-dropdown.scss';
@import '_toggle_button.scss';
