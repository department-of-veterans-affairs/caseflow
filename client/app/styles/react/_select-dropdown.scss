$cf-select-bg-color: #fff;
$cf-select-disabled-bg-color: $color-gray-lightest;
$cf-select-disabled-color: #323a45;
$cf-select-border-color: #5b616b;
$cf-select-text-color: #323a45;

.cf-select {
  .cf-select__control {
    background-color: $cf-select-bg-color;
    border: 1px solid $cf-select-border-color;
    border-collapse: separate;
    border-radius: 0;
    border-spacing: 0;
    color: $cf-select-text-color;
    cursor: default;

    &--is-disabled {
      background-color: $cf-select-disabled-bg-color;

      .cf-select__single-value {
        color: $cf-select-disabled-color;
      }

      .cf-select__indicators {
        opacity: 0.35;
      }

      input {
        margin: 0;
      }
    }

    .cf-select__value-container {
      padding-bottom: 0;
      padding-top: 0;

      .cf-select__placeholder {
        color: $cf-select-text-color;
      }

      // Ensure that the "selected" value is aligned correctly
      .cf-select__placeholder,
      .cf-select__single-value {
        position: unset;
        top: unset;
        transform: unset;
      }
    }

    .cf-select__indicators {
      .cf-select__indicator-separator {
        background-color: $cf-select-border-color;
        margin-bottom: 0;
        margin-top: 0;
      }

      .cf-select__indicator {
        svg {
          fill: $cf-select-text-color;
        }
      }
    }
  }

  .cf-select__menu {
    border-radius: 0;

    .cf-select__menu-list {
      .cf-select__option {
        background-color: $color-white;
        color: $color-gray;

        &.cf-select__option--is-focused {
          background-color: $color-gray;
          color: $color-white;
        }
      }
    }
  }

  .usa-input-error & {
    .cf-select--is-disabled {
      input { width: 1px; }
    }
  }
}
