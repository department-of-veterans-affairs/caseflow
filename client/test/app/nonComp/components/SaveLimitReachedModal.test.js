import React from 'react';
import { Provider } from 'react-redux';
import { axe } from 'jest-axe';
import { MemoryRouter as Router } from 'react-router-dom';
import SaveLimitReachModal from 'app/nonComp/components/ReportPage/SaveLimitReachedModal';
import { render, screen } from '@testing-library/react';

import createNonCompStore from 'test/app/nonComp/nonCompStoreCreator';
import COPY from 'app/../COPY';

describe('SaveLimitReachModal', () => {
  const setup = (storeValues = {}) => {
    const store = createNonCompStore(storeValues);

    return render(
      <Provider store={store}>
        <Router>
          <SaveLimitReachModal />
        </Router>
      </Provider>
    );
  };

  describe('renders correctly', () => {
    it('passes a11y testing', async () => {
      const { container } = setup();

      const results = await axe(container);

      expect(results).toHaveNoViolations();
    });

    it('renders correctly', () => {
      const { container } = setup();

      expect(container).toMatchSnapshot();
    });
  });

  describe('displays header and description properly', () => {
    it('should display title and description of the modal', () => {
      setup();

      expect(screen.getByText('Limit Reached')).toBeTruthy();

      expect(screen.getByText(COPY.SAVE_LIMIT_REACH_MESSAGE)).toBeTruthy();
    });
  });

});
