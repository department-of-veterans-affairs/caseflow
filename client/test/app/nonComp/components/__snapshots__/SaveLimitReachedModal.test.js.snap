// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SaveLimitReachModal renders correctly renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        aria-describedby="modal_id-desc"
        aria-labelledby="modal_id-title"
        aria-modal="true"
        class="cf-modal active "
        id="modal_id"
        role="dialog"
      >
        <div
          class="cf-modal-body"
          id=""
        >
          <button
            class="cf-modal-close"
            id="Limit-Reached-button-id-close"
            type="button"
          >
            <span
              class="usa-sr-only"
            >
              Close
            </span>
            <svg
              class="cf-icon-close"
              height="55"
              viewBox="0 0 55 55"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                close
              </title>
              <g
                fill="#212121"
              >
                <path
                  d="M52.6 46.9l-6 6c-.8.8-1.9 1.2-3 1.2s-2.2-.4-3-1.2l-13-13-13 13c-.8.8-1.9 1.2-3 1.2s-2.2-.4-3-1.2l-6-6c-.8-.8-1.2-1.9-1.2-3s.4-2.2 1.2-3l13-13-13-13c-.8-.8-1.2-1.9-1.2-3s.4-2.2 1.2-3l6-6c.8-.8 1.9-1.2 3-1.2s2.2.4 3 1.2l13 13 13-13c.8-.8 1.9-1.2 3-1.2s2.2.4 3 1.2l6 6c.8.8 1.2 1.9 1.2 3s-.4 2.2-1.2 3l-13 13 13 13c.8.8 1.2 1.9 1.2 3s-.4 2.2-1.2 3z"
                />
              </g>
            </svg>
          </button>
          <div
            style="display: flex; flex-wrap: wrap;"
          >
            <h1
              id="modal_id-title"
              style="flex: 1;"
            >
              Limit Reached
            </h1>
            <div
              data-css-2flkvz=""
            >
              You've reached the max limit of 10 saved searches. Please select a search to delete before proceeding.
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_10"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="10"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_10"
                    >
                      Search Name2
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_11"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="11"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_11"
                    >
                      Search Name3
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_12"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="12"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_12"
                    >
                      Search Name4
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_13"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="13"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_13"
                    >
                      Search Name4
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_14"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="14"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_14"
                    >
                      Search Name4
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_15"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="15"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_15"
                    >
                      Search Name4
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_16"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="16"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_16"
                    >
                      Search Name4
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_17"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="17"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_17"
                    >
                      Search Name2
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_18"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="18"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_18"
                    >
                      Search Name2
                    </label>
                  </div>
                </div>
              </fieldset>
              <fieldset
                class="usa-fieldset-inputs cf-form-radio "
              >
                <legend
                  class="usa-sr-only"
                >
                  <span>
                    radioFieldSearchGroup
                     
                  </span>
                </legend>
                <div
                  class="cf-form-radio-options"
                  style="margin-left: 5px;"
                >
                  <div
                    class="cf-form-radio-option"
                  >
                    <input
                      id="radioFieldSearchGroup_20"
                      name="radioFieldSearchGroup"
                      type="radio"
                      value="20"
                    />
                    <label
                      class=""
                      for="radioFieldSearchGroup_20"
                    >
                      Search Name2
                    </label>
                  </div>
                </div>
              </fieldset>
            </div>
          </div>
          <div
            class="cf-modal-divider"
          />
          <div
            class="cf-modal-controls"
          >
            <span>
              <button
                class="cf-modal-link cf-btn-link cf-push-left usa-button"
                id="Limit-Reached-button-id-0"
                type="button"
              >
                Cancel
              </button>
            </span>
            <span>
              <button
                class="cf_add_margin cf-push-right usa-button-disabled usa-button"
                disabled=""
                id="Limit-Reached-button-id-1"
                type="button"
              >
                Delete
              </button>
            </span>
            <span>
              <button
                class="usa-button usa-button-secondary cf-push-right usa-button"
                id="Limit-Reached-button-id-2"
                type="button"
              >
                View saved searches
              </button>
            </span>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      aria-describedby="modal_id-desc"
      aria-labelledby="modal_id-title"
      aria-modal="true"
      class="cf-modal active "
      id="modal_id"
      role="dialog"
    >
      <div
        class="cf-modal-body"
        id=""
      >
        <button
          class="cf-modal-close"
          id="Limit-Reached-button-id-close"
          type="button"
        >
          <span
            class="usa-sr-only"
          >
            Close
          </span>
          <svg
            class="cf-icon-close"
            height="55"
            viewBox="0 0 55 55"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              close
            </title>
            <g
              fill="#212121"
            >
              <path
                d="M52.6 46.9l-6 6c-.8.8-1.9 1.2-3 1.2s-2.2-.4-3-1.2l-13-13-13 13c-.8.8-1.9 1.2-3 1.2s-2.2-.4-3-1.2l-6-6c-.8-.8-1.2-1.9-1.2-3s.4-2.2 1.2-3l13-13-13-13c-.8-.8-1.2-1.9-1.2-3s.4-2.2 1.2-3l6-6c.8-.8 1.9-1.2 3-1.2s2.2.4 3 1.2l13 13 13-13c.8-.8 1.9-1.2 3-1.2s2.2.4 3 1.2l6 6c.8.8 1.2 1.9 1.2 3s-.4 2.2-1.2 3l-13 13 13 13c.8.8 1.2 1.9 1.2 3s-.4 2.2-1.2 3z"
              />
            </g>
          </svg>
        </button>
        <div
          style="display: flex; flex-wrap: wrap;"
        >
          <h1
            id="modal_id-title"
            style="flex: 1;"
          >
            Limit Reached
          </h1>
          <div
            data-css-2flkvz=""
          >
            You've reached the max limit of 10 saved searches. Please select a search to delete before proceeding.
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_10"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="10"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_10"
                  >
                    Search Name2
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_11"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="11"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_11"
                  >
                    Search Name3
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_12"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="12"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_12"
                  >
                    Search Name4
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_13"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="13"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_13"
                  >
                    Search Name4
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_14"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="14"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_14"
                  >
                    Search Name4
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_15"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="15"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_15"
                  >
                    Search Name4
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_16"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="16"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_16"
                  >
                    Search Name4
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_17"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="17"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_17"
                  >
                    Search Name2
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_18"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="18"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_18"
                  >
                    Search Name2
                  </label>
                </div>
              </div>
            </fieldset>
            <fieldset
              class="usa-fieldset-inputs cf-form-radio "
            >
              <legend
                class="usa-sr-only"
              >
                <span>
                  radioFieldSearchGroup
                   
                </span>
              </legend>
              <div
                class="cf-form-radio-options"
                style="margin-left: 5px;"
              >
                <div
                  class="cf-form-radio-option"
                >
                  <input
                    id="radioFieldSearchGroup_20"
                    name="radioFieldSearchGroup"
                    type="radio"
                    value="20"
                  />
                  <label
                    class=""
                    for="radioFieldSearchGroup_20"
                  >
                    Search Name2
                  </label>
                </div>
              </div>
            </fieldset>
          </div>
        </div>
        <div
          class="cf-modal-divider"
        />
        <div
          class="cf-modal-controls"
        >
          <span>
            <button
              class="cf-modal-link cf-btn-link cf-push-left usa-button"
              id="Limit-Reached-button-id-0"
              type="button"
            >
              Cancel
            </button>
          </span>
          <span>
            <button
              class="cf_add_margin cf-push-right usa-button-disabled usa-button"
              disabled=""
              id="Limit-Reached-button-id-1"
              type="button"
            >
              Delete
            </button>
          </span>
          <span>
            <button
              class="usa-button usa-button-secondary cf-push-right usa-button"
              id="Limit-Reached-button-id-2"
              type="button"
            >
              View saved searches
            </button>
          </span>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
